website/client/src/components/tasks/spells.vue

```diff
@@ -1,26 +1,18 @@
 <template>
   <div v-if="user.stats.lvl > 10">
-    <div
-      v-if="potionClickMode"
-      ref="clickPotionInfo"
-      class="dragInfo mouse"
-    >
-      <div class="spell col-12 row">
-        <div class="col-8 details">
-          <p class="title">
-            {{ spell.text() }}
-          </p>
+    <div v-if="potionClickMode" ref="clickPotionInfo" class="dragInfo mouse">
+      <div class="spell">
+        <div class='spell-border'>
+          <div class="mana">
+            <div class="img" :class="`shop_${spell.key} shop-sprite item-img`"></div>
+          </div>
+        </div>
+        <div class="details">
           <p class="notes">
-            {{ `Click on a ${spell.target} to cast!` }}
+            {{ "Select a task" }}
```

Comment:
Can we revert this to the original text `{{ `Click on a ${spell.target} to cast!` }}` because it could also target a party member instead of a task

Reply:
@paglias It will affect your given length of the card. So if its ok for you then i can do it

Reply:
![2020-11-24 (2)](https://user-images.githubusercontent.com/44499187/100083013-ecdb3000-2e6e-11eb-830c-929e8fe31967.png)
I am talking about the dark box below it.
Either i can increase the box height and make them text align center or i can increase the length and  make the complete text side by side.